frontend/src/components/accounts/AccountDetails.tsx

```diff
@@ -119,9 +120,13 @@ export default class extends React.Component<Props, State> {
             <CardCell
               title="Creation Hash"
               text={
-                <TransactionLink transactionHash={account.address}>
-                  {account.address}
-                </TransactionLink>
+                account.address === message ? (
```

Comment:
We should just make that string empty wherever we create a "dummy" account object, and check if it is an empty string here and then display a human-readable message here.